### ISSUE TITLE
Can you interested in test a mini 3D scanner？

### ISSUE THREAD

**Issue by huminlia** [zjudmd1015/Mini-3D-Scanner#8]
A [handheld 3D scanner](http://www.revopoint3d.com), its' accuracy is up to 0.3mm. You can put it in your pocket

**Comment by Murray1199** [comment]
Revopoint's New POP 3 [3D Scanner](https://www.revopoint3d.com/). Eliminate the need to manually measure objects by consistently and efficiently capturing accurate and detailed 3D models at fast scanning speeds between 12 to 18fps with a single-frame precision of up to 0.05mm and 0.05mm point distance.

**Comment by miachelbrown** [comment]
Yes, I have purchased a [3D scanner](https://www.revopoint3d.com/) from the Mini series produced by Revopoint Company. Its primary application is to scan small objects. Compared to other series of products, the Mini series excels in capturing details and precision for small items. Moreover, the Mini scanner is lightweight and highly portable, making it easy to carry, and it can be connected and used with a smartphone outdoors.

**Comment by miachelbrown** [comment]
Certainly! I'd be excited to test a mini 3D scanner. In fact, I've had the opportunity to use the Revopoint MINI along with its user-friendly 3D scanner app, and I must say, the experience has been quite impressive.
The Revopoint MINI is a compact and versatile 3D scanner that offers a range of applications. Its user-friendly design and portability make it a convenient tool for capturing detailed 3D models on the go. I found the scanning process through the dedicated 3D scanner app to be intuitive and straightforward, and the results were surprisingly accurate.
One of the standout features of the Revopoint MINI is its ability to scan small objects with intricate details. It's been a game-changer for me when it comes to digitizing objects for various projects, whether it's for 3D printing, digital design, or even documentation.
The scanning software, which is an essential part of the package, enhances the user experience by providing real-time feedback and interactive guidance. This app allows you to see the scanning progress, make adjustments as needed, and conveniently export the resulting 3D models to various file formats for further use in other applications.
Overall, I've been quite impressed with the Revopoint MINI's capabilities and the seamless integration between the device and its accompanying [3D scanner app](https://www.revopoint3d.com/download/). It's made 3D scanning accessible and convenient, especially for those who want to explore the world of 3D without the complexity of larger scanners. If you're looking for a versatile and effective mini 3D scanner that comes with an easy-to-use app, the Revopoint MINI is definitely worth considering.

**Comment by OliviaDavis-art** [comment]
I am pleased to inform you that [Revopoint](https://www.revopoint3d.com/) has now introduced the all-new integrated 3D scanner, [MIRACO.](https://www.revopoint3d.com/products/standalone-3d-scanner-miraco) In my opinion, MIRACO ingeniously combines the features of Revopoint's previous products, MINI and RANGE, allowing for the scanning of objects of various sizes. MIRACO integrates scanning and editing functions, making it incredibly convenient for users. Additionally, it is compact, with a size comparable to that of a camera and even lighter in weight. The two scanning modes cater to a variety of needs, making it truly impressive in my view.
